src/current/v24.3/data-resilience.md

```diff
@@ -0,0 +1,110 @@
+- [**Fault tolerance**]({% link {{ page.version.version }}/demo-fault-tolerance-and-recovery.md %}): Capabilities built in to CockroachDB to perform routine maintenance operations without any affect on performance. For example, [online schema changes]({% link {{ page.version.version }}/online-schema-changes.md %}), [WAL failover]({% link {{ page.version.version }}/cockroach-start.md %}#write-ahead-log-wal-failover).
+- **Logical data replication (LDR)** (Preview): A cross-cluster replication tool between multiple active CockroachDB clusters, which supports a diverse range of topologies. LDR provides eventually consistent, table-level replication.
+
+### Choose a high availability strategy
+
+CockroachDB uses synchronous, built-in Raft replication to create and distribute copies of data, ensuring consistency across the data copies. The database can tolerate nodes going offline without service interruption, whether in a single-region or [multi-region]({% link {{ page.version.version }}/multiregion-overview.md %}) cluster.
+
+<table class="comparison-chart">
+  <tr>
+    <th></th>
+    <th>Single-region replication (synchronous)</th>
+    <th>Multi-region replication (synchronous)</th>
+    <th>Logical data replication (asynchronous)</th>
+  </tr>
+
+  <tr>
+    <td class="comparison-chart__feature">
+      <b>RPO</b>
+    </td>
+    <td>0 seconds</td>
+    <td>0 seconds</td>
+    <td>Immediate mode 0.5 seconds<br>Transactional mode ~10s of seconds</td>
+  </tr>
+
+  <tr>
+    <td class="comparison-chart__feature">
+      <b>RTO</b>
+    </td>
+    <td>Zero RTO<br>Potential increased latency for 1-9 seconds</td>
+    <td>Zero RTO<br>Potential increased latency for 1-9 seconds</td>
+    <td>Zero RTO<br>Add application cutover time</td>
+  </tr>
+
+  <tr>
+    <td class="comparison-chart__feature">
+      <b>Write latency</b>
+    </td>
+    <td>Region-local write latency<br>p50 latency < 5ms (for multiple availability zones in us-east1)</td>
+    <td>Cross-region write latency<br>p50 latency > 50ms  (for a multi-region cluster in us-east1, us-east-2, us-west-1)</td>
+    <td>Region-local latency depending on design<br>p50 latency < 5ms (for multiple availability zones in us-east1)</td>
+  </tr>
+
+  <tr>
+    <td class="comparison-chart__feature">
+      <b>Recovery</b>
+    </td>
+    <td>Automatic</td>
+    <td>Automatic</td>
+    <td>Semi-automatic</td>
+  </tr>
+
+  <tr>
+    <td class="comparison-chart__feature">
+      <b>Hardware cost</b>
+    </td>
+    <td>1x<br>Networking between availability zones</td>
+    <td>1x<br>Networking between datacenters</td>
+    <td>2x<br>Networking between availability zones or datacenters</td>
+  </tr>
```

Comment:
I don't think the 1x and 2x are clear here. Perhaps we need an explanation as a footnote at the bottom of the table?

Reply:
I'll look into this as I'm doing the PM feedback round.

Reply:
I think we should delete hardware cost lineitem.

Reply:
I have deleted the hardware line items for now.